receiver/kubeletstatsreceiver/internal/kubelet/metadata_test.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// nolint:errcheck
```

Comment:
Why no check?

Reply:
somehow the IDE inserted and missed it.
Fixed